apps/web/app/hooks/features/useAuthTeamTasks.ts

```
@@ -4,64 +4,60 @@ import { useMemo } from 'react';
+		).totalTasks;
+
+		const todayTasksCOunt = getTotalTasks(todayPlan);
+
+		const futureTasksCount = getTotalTasks(futurePlans);
+
+		return outStandingTasksCount + futureTasksCount + todayTasksCOunt;
+	}, [futurePlans, outstandingPlans, todayPlan]);
+
+	const totalTodayTasks = useMemo(
+		() =>
+			currentMember?.totalTodayTasks && currentMember?.totalTodayTasks.length
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)